Library/Homebrew/cask/audit.rb

```diff
@@ -342,6 +345,75 @@ def bad_osdn_url?
       bad_url_format?(/osd/, [%r{\Ahttps?://([^/]+.)?dl\.osdn\.jp/}])
     end
 
+    def homepage
+      URI(cask.homepage.to_s).host
+    end
+
+    def domain
+      URI(cask.url.to_s).host
+    end
+
+    def url_match_homepage?
+      host = cask.url.to_s.downcase
+      host_uri = URI(host)
+      host = if host.match?(/:\d/) && host_uri.port != 80
+        "#{host_uri.host}:#{host_uri.port}"
+      else
+        host_uri.host
+      end
+      home = homepage.downcase
+      if (split_host = host.split(".")).length >= 3
+        host = split_host[-2..].join(".")
+      end
+      if (split_home = homepage.split(".")).length >= 3
+        home = split_home[-2..].join(".")
+      end
+      host == home
+    end
+
+    def strip_url_scheme(url)
+      url.sub(%r{^.*://(www\.)?}, "")
+    end
+
+    def url_from_verified
+      cask.url.verified.sub(/[^ ]*# ([^ ]+) .*/, '\1').sub(%r{^https?://}, "")
+    end
+
+    def verified_matches_url?
+      strip_url_scheme(cask.url.to_s).include?(url_from_verified)
+    end
+
+    def verified?
```

Comment:
Naming this `verified?` makes it sound like it is actually verified correctly instead of `verified:` only being present.

Reply:
Rename as `verified_present?`.